wurf_runner.py

```diff
@@ -158,6 +157,19 @@ def make_run(taskgen, run_type):
         test_files = getattr(taskgen, 'test_files', [])
         task.test_inputs = taskgen.to_nodes(test_files)
 
+        # Locate the shared libs this binary is depending on (if any).
+        # And Add these to the set of test inputs.
+        for lib in taskgen.uselib:
+            lib = taskgen.bld.task_gen_cache_names.get(lib, None)
```

Comment:
What is the type of `lib` I think that would be good to document here. Will make it easier to find the relevant code sometime in the future.